### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by MrNuke007** [fsinfuhh/Bitpoll#148]
Uh this is a test, how do I create a folder ?

**Comment by ftsell** [comment]
I'm not sure what you're intending with this issue. Could you rephrase?

**Comment by MrNuke007** [comment]
Where can I publish my code on this app ?

**Comment by MrNuke007** [comment]
There's no tutorial on how to use this

**Comment by ftsell** [comment]
Do you mean GitHub? Or the Bitpoll app?

Because Bitpoll is an app for doing polls and has nothing to do with sharing code.
You've opened an issue in the Bitpoll repository which is usually done if you are having a problem with Bitpoll or wish to request a new feature.

This repository however is hosted on GitHub which is a collaborative coding platform.
I suggest you look up some tutorials on how to use GitHub. There are many YouTube videos explaining how git and GitHub work.

**Comment by MrNuke007** [comment]
Well, I don't really know how to use this, I saw a lot of people talking about it.
